java/ql/test/experimental/query-tests/security/CWE-400/ThreadResourceAbuse.java

```diff
@@ -105,4 +149,47 @@ protected void doHead(HttpServletRequest request, HttpServletResponse response)
 			}
 		}
 	}
+
+	int parseReplyAfter(String value) {
```

Comment:
Really minor, but shouldn't this be <code>parseRe<del>pl</del><ins>tr</ins>yAfter</code>?

Reply:
Implementation of the linked CVE has the method name <code>parseReplyAfter</code> so I used  it. However, the name <code>parseRetryAfter</code> is more accurate so I've made the change.